seccomp.go

```diff
@@ -171,6 +171,8 @@ const (
 	ArchPARISC64
 	// ArchRISCV64 represents RISCV64
 	ArchRISCV64
+	// ArchLOONGARCH64 represents 64-bit LoongArch.
+	ArchLOONGARCH64
```

Comment:
`ArchM68K`, `ArchSH`, and `ArchSHEB` have to be added as well

Reply:
Marking this as resolved in the latest revision.